src/app/_common/monitoring/amplitude.js

```diff
@@ -45,25 +45,22 @@ export const logSearchFilterRemoved = (data) => {
 
 function getJobPostingFormat(jobPosting) {
     if (
-        jobPosting &&
-        jobPosting._source &&
-        jobPosting._source.properties &&
-        jobPosting._source.properties.adtext &&
-        jobPosting._source.properties.adtext.includes('<section id="arb-serEtter">') &&
-        jobPosting._source.properties.adtext.includes('<section id="arb-arbeidsoppgaver">') &&
-        jobPosting._source.properties.adtext.includes('<section id="arb-tilbyr">')
+        adData.adText &&
+        adData.adText.includes('<section id="arb-serEtter">') &&
+        adData.adText.includes('<section id="arb-arbeidsoppgaver">') &&
+        adData.adText.includes('<section id="arb-tilbyr">')
     ) {
         return JobPostingTextEnum.STRUKTURERT;
     }
     return JobPostingTextEnum.IKKE_STRUKTURERT;
 }
 
-export function logStillingVisning(ad) {
+export function logStillingVisning(adData) {
     // Todo - tror employer.location er erstattet med employer.locationList
-    const employerLocation = ad._source.employer ? ad._source.employer.location : null;
+    const employerLocation = adData.employer && adData.employer.locationList ? adData.employer.locationList[0] : null;
```

Comment:
Samma som ovan. Flytta hellre in logiken i AdData och exponera ett fält `employerLocation`.

Reply:
Employer Location er tricky. Blir brukt på forskjellige måter på forskjellige steder så da tenker jeg heller at det er opptil de forskjellige komponentene til å formatere data. F.eks.

Her bruker amplitude den første location den ser. Mens andre steder som EmployerDetails komponentet så looper vi gjennom alle locationList og kombinerer dem inn til en string. Tenker det er bedre da å beholde de "fundementale" data som diverse steder kan formatere til sitt eget.